.github/workflows/tests.yml

```diff
@@ -44,3 +44,12 @@ jobs:
         with:
           name: dist
           path: dist/
+  title:
+    name: Check PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ExpediaGroup/github-helpers@v1
+        with:
+          helper: check-pr-title
+          pull_number: ${{ github.event.pull_request.number }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We already have this workflow setup in the repo actually

Reply:
oh lol i hadn't pulled the latest from `main`, i'll remove this